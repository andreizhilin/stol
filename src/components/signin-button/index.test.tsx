import { render } from '@testing-library/react';
import { describe, it, expect } from 'vitest';

import { SigninButton } from '.';

describe('<SigninButton />', () => {
  it('renders correctly', async () => {
    const { asFragment } = render(<SigninButton />);
    expect(asFragment()).toMatchSnapshot();
  });
});
