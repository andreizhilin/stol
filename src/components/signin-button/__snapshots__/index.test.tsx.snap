// Vitest Snapshot v1

exports[`<SigninButton /> > renders correctly 1`] = `
<DocumentFragment>
  <a
    class="cursor-pointer"
    href="https://oauth.yandex.ru/authorize?response_type=token&client_id=1bf73c3b9e5d4332a7f1d7452fb57084&redirect_uri="
  >
    Вход
  </a>
</DocumentFragment>
`;
